src/view/drag-handle/util/should-allow-dragging-from-target.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+import type { Props } from '../drag-handle-types';
+
+export const interactiveTagNames: string[] = [
```

Comment:
Add `audio`, `video` and `optgroup` as per sean's suggestion

Reply:
done :)